src/Core/Models/Domain/Cipher.cs

```diff
@@ -115,7 +117,7 @@ public async Task<CipherView> DecryptAsync()
             switch (Type)
             {
                 case Enums.CipherType.Login:
-                    model.Login = await Login.DecryptAsync(OrganizationId, model.Key);
+                    model.Login = await Login.DecryptAsync(OrganizationId, bypassValidation, model.Key);
```

Comment:
🤔 Is the `bypassValidation` needed here? Couldn't we just pass:
```suggestion
                    model.Login = await Login.DecryptAsync(OrganizationId, Key != null, model.Key);
```
Furthermore, I'm not 100% sure, but maybe you can skip the parameter and just check inside the method if `model.Key` is `null` or not.
If it's needed I'd change the name here to be more explicit of what this validation is referring to: `bypassUriChecksumValidation`.

Reply:
I believe there are cases where the `model.key` is different than `null` but isn't a `Key` from IVIE, so we would get false positives.

Reply:
alright then changing the name would suffice in this case 😄